samples/trivia/trivia-shared/src/commonMain/kotlin/app/cash/zipline/samples/trivia/trivia.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2022 Block, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.zipline.samples.trivia
+
+import app.cash.zipline.ZiplineService
+import kotlinx.serialization.Serializable
+
+interface TriviaService : ZiplineService {
+  fun games(): List<TriviaGame>
+  fun answer(gameId: Int, questionId: Int, answer: String): AnswerResult
+}
+
+@Serializable
+class TriviaGame(
+  val id: Int,
+  val name: String,
+  val questions: List<Question>,
+)
+
+@Serializable
+class Question(
+  val id: Int,
+  val text: String,
+)
+
+@Serializable
+class AnswerResult(
+  val correct: Boolean,
+  val message: String,
+)
```

Comment:
Do you think making this a `sealed class` (or `sealed interface` if they work?) would better indicate that it's flexible enough to deal cases other than a standard `class`? I'm just guessing that skimming this may give the incorrect assumption that we can only pass a standard `class` over the bridge.

```kotlin
sealed class Answer {

  abstract val message: String

  data class Correct(override val message: String): Answer()

  data class Wrong(override val message: String): Answer()
}
```

Reply:
I wanna keep this particular example as small as possible, and then maybe do recipes to talk through all the different features. It’s tempting to build out flows and other features into this example!